packaging/before-packaging-command/src/main.rs

```diff
@@ -171,13 +171,21 @@ fn download_wasmedge_macos(version: &str) -> std::io::Result<()> {
     // Command 1: Create the destination directory.
     let dest_dir = std::env::current_dir()?.join("wasmedge");
     fs::create_dir_all(&dest_dir)?;
-
+    let mut wasmedge_download_url = String::new();
+    let mut wasi_nn_download_url = String::new();
+    if std::env::consts::ARCH == "x86_64" {
```

Comment:
Don't we need to check the *target* architecture, not the architecture of the build machine? This assumes we never do any cross compilation and only build for the current arch.

Reply:
Or, at the very least, allow it to be overridden with a CLI arg or env var.